cmd.js

```diff
@@ -2,4 +2,10 @@
 
 'use strict'
 
-require('fastify-cli/generate').cli(['.', ...process.argv.slice(2)])
+const args = process.argv.slice(2)
```

Comment:
```suggestion
const args = [...process.argv.slice(2)]
```

(avoid to mutate `process.argv`)

Reply:
I can do this change, but slice does produce a [shallow copy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice), so the existing implementation wouldn't mutate `process.argv`.

```
$ node
Welcome to Node.js v20.11.1.
Type ".help" for more information.
> process.argv = [1,2,3,4]
[ 1, 2, 3, 4 ]
> const args = process.argv.slice(2)
undefined
> args
[ 3, 4 ]
> args.unshift('hello')
3
> args
[ 'hello', 3, 4 ]
> process.argv
[ 1, 2, 3, 4 ]
> args.push('world')
4
> args
[ 'hello', 3, 4, 'world' ]
> process.argv
[ 1, 2, 3, 4 ]
```